src/Common/Infrastructure/Controllers/RegistryApiController.cs

```diff
@@ -36,7 +36,13 @@ public abstract class RegistryApiController<T> : ApiController<T>
             ILogger<T> logger) : base(redis, logger)
         {
             _restClient = restClient;
-            CacheToggle = cacheToggle;
+            _cacheToggle = cacheToggle;
+        }
+
+        protected bool ShouldGetFromCache()
```

Comment:
@ArneD we're not yet convinced about the method name. The default behavior should be that the cache is tried. The default behavior can than be changed through config our a client's request. We were doubting between CanGetFromCache or the current name.